test/extended/networking/egressip.go

```diff
@@ -154,11 +154,6 @@ var _ = g.Describe("[sig-network][Feature:EgressIP]", func() {
 			targetPort)
 
 		g.By("Creating a project for the prober pod")
-		// Create a target project and assign source and target namespace
```

Comment:
nit: can you keep:
~~~
	// Create a target project and assign source and target namespace
Attach files by dragging & dropping, selecting or pasting them.
		// to variables for later use.
~~~